api/views.py

```diff
@@ -26,6 +27,9 @@
 
 logger = logging.getLogger('gutenberg.api.printing')
 
+def Error(error, message):
+    return {"error": error, "message": message}
+
```

Comment:
The purpose of the `Error.error` field is unclear to me. Does it always refer to a field name in the input request? If so, use a more descriptive name